src/controller/instructor/instructor.controller.ts

```diff
@@ -44,4 +44,49 @@ import {
       const res = await this.instructorService.allCourses(req.user[0].id);
       return res;
     }
+
+
+    @Get('/getAllUpcomingClasses')
+  @ApiOperation({ summary: 'Get all upcoming classes by instructorId' })
```

Comment:
Can we add another query with the **status** ["all", 'upcoming', "ongoing", "completed"], as well as other query strings?

In status, if we pass all, it will fetch all the classes sorted by the start time of the class. if we pass the upcoming only then it will return only to the upcoming class. so on.